impeller/renderer/backend/metal/formats_mtl.mm

```diff
@@ -64,8 +64,13 @@
 
   auto des = [[MTLDepthStencilDescriptor alloc] init];
 
-  des.depthCompareFunction = ToMTLCompareFunction(depth->depth_compare);
-  des.depthWriteEnabled = depth->depth_write_enabled;
+  // These temporary variables are necessary for clang-tidy (Fuchsia LLVM
```

Comment:
I don't see the crash that this fixes happening on CI. Could we maybe split this out into a separate issue/PR?

Reply:
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8779203752399309425/+/u/test:_lint_host_debug/stdout?format=raw

<img width="1015" alt="Screenshot 2023-06-05 at 19 11 32" src="https://github.com/flutter/engine/assets/96958/23b4ebaf-da62-4845-a2ca-039b416002fc">

This is from https://github.com/flutter/engine/pull/42399

I can try to split the PR.

Reply:
Oh, yeah, sorry. I see it now.